packages/docs/verite/patterns/travel-rule.md

```diff
@@ -2,53 +2,104 @@
+3. Bob's financial institution verifies the originator's proof-of-control.
+4. Bob's financial institution constructs message "M2" which is a VC proving control over the beneficiary address.
+5. Bob's FI transmits M2 privately to Alice's FI using the callback URL specified in M1.
+6. Alice's FI receives and verifies the VC in M2, leaving both Alice's FI and Bob's FI (a) certain that each controls the designated addresses and (b) with a private secure network connection established through the specified callback URL.
+
 
-Before the transaction is broadcast, the sending and receiving FIs use the chain-agnostic mempool to publish and observe messages establishing and requesting proof of address control. These messages contain no customer data and no specific transaction data. Since the transaction amount is not included, observers are unable to determine information about the planned transaction other than its intended origin and destination (this is less information than is available in a typical mem pool, in which the actual monetary transaction is broadcast). This allows counterparty endpoints to find and connect to one another privately before exchanging counterparty information, and the proof of address credentials prevent other parties from spoofing the originator or beneficiary.
+### Part 3: Exchange of Counterparty Credentials
 
-### Part 2: Exchange of Counterparty Credentials
+If the transaction does not meet the threshold required for the Travel Rule, then Part 3 is not necessary and does not apply, as Alice's FI can simply proceed to broadcasting the blockchain transaction. 
+
+If the Travel Rule does apply, however, both financial institutions are able to exchange counterparty verifiable credentials over callback URLs that allow for private off-chain secure communication. This can happen over any secure channel, and today is mostly done in consortium/network-like platforms that bundle the authentication of step 2 and the secure channel for step 3 (as well as data dictionaries or translation enginges for exchanging the actual information required by applicable reporting requirements, itself a massively complex value-add and rules engine).  This research could be called long-tail, as the rapid pace of regulatory change here means the existing protocols will probably be distracted updating their rules engines for some time and not have bandwidth to consider harmonizing on UX and consent flows!
```

Comment:
I would remove "private off-chain secure communication" with "private secure comunication", since the bulletin board maybe not be necessary on-chain